test/regression/test_deploy.py

```diff
@@ -1,13 +1,46 @@
 #!/usr/bin/python3
 
 import argparse
+import re
+import subprocess # nosec
 import sys
 
+import yaml
+
 from checker import Checker
 from cluster import Cluster
 from utils import Logger, StopWatch
 
 
+def verify_pods_in_namespace(checker, namespace):
+    stdout = subprocess.Popen(['kubectl', 'get', 'pods', '-A'], # nosec
```

Comment:
This function is `verify_pods_in_namespace`, but then the command you use to get pods is using the `-A` flag which gets all pods across all namespaces. Should this command be updated to use `-n namespace` instead?

Reply:
Good point.... not sure why I would have done it this way.  🤔

Reply:
Ok, I fixed it.